streamlit_web.py

```diff
@@ -651,6 +654,49 @@ def interactive():
         summaries = sz.summarize_text(input_text)
         st.write(summaries)
 
+def senti_pos_iter(tokens_column):
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        # Start off with an empty words list to add to over time
+        words = []
+        # If the word isn't in the words list already, then it's added to the words list
+        for word in token_element:
+            if word not in words:
+                words.append(word)
```

Comment:
Another suggestion: since the tokens are already in a list, in the data frame column. Is it possible to just return the unique elements without going into a for loop?

Reply:
Also, can you just explain more why is needed to append every token to the `words` list, instead of just directly using the token lists.

Reply:
The idea is that the words list would have unique elements. You would have to go through the tokens list anyway to delete the repeating words, so the performance wouldn't change by that much. If you have the most subjective word being repeated, you would have the top three words being the same word.

Reply:
Okay that's what I thought. So it's probably worth looking into methods that would return unique elements in a list. There are methods for that, without having to go into a loop

Reply:
I'll try converting to a set